tests/unit/test_current_injection.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import numpy
+
+
+@pytest.mark.parametrize("create_tmp_simulation_config_file", [
+    {
+        "simconfig_fixture": "ringtest_baseconfig",
+        "extra_config": {
+            "inputs": {
+                "Stimulus": {
+                    "module": "pulse",
+                    "input_type": "current_clamp",
+                    "delay": 1,
```

Comment:
I would avoid to use 1s as they are often default values. In addition if something multiplies we cannot distinguish if it multiplies or it replaces:

`a = b` is the same as `a = 1 * b`